### PR TITLE
doc: update readme file

### PR DESCRIPTION
**Pull request by davdarras**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,194 +22,218 @@ mvn spring-boot:run
 ```  
 
 ## Application Accesses locally
-To access to swagger-ui, use this url : [http://localhost:8080/api/swagger-ui.html](http://localhost:8080/swagger-ui.html)  
-To access to h2 console, use this url : [http://localhost:8080/api/h2-console](http://localhost:8080/api/h2-console)  
+To access to swagger-ui, use this url : [http://localhost:8080/api/swagger-ui/index.html](http://localhost:8080/swagger-ui/index.html)
```
Comment (nicolasTurban) [inline README.md:25]: Duplicate of line 42 ?

File: README.md [hunk 2]
```diff
@@ -22,194 +22,218 @@ mvn spring-boot:run
...
+logging:
+  file:
+    name: /path/to/log/folder/queen.log
+  level:
+    root: INFO
+springdoc:
+  swagger-ui:
+    oauth:
+      additionalQueryStringParams:
+        kc_idp_hint : idp_hint
```
Comment (nicolasTurban) [inline README.md:149]: Specific to keycloak implementation ?

Comment (davdarras) [reply README.md]: deleted :)

File: README.md (hunk 1 above)
Comment (davdarras) [reply README.md]: deleted :)

**Pull request merged**